src/song_utils.py

```diff
@@ -8,12 +8,43 @@
 from mutagen.id3 import ID3FileType, PictureType
 from mutagen.ogg import OggFileType
 from mutagen.wave import WAVE
-from nextcord import File
+from nextcord import Attachment, Embed, File, Member, User
+from nextcord.utils import escape_markdown
 from PIL import Image, UnidentifiedImageError
 
 import config
 
 
+def embed_song(
+    message_content: str,
+    attachment_filepath: str,
+    attachment: Attachment,
+    user: User | Member,
```

Comment:
Member is a subclass of user so there is no need for a union type. Choose the most general type applicable